src/main/java/co/rsk/federate/BtcToRskClient.java

```diff
@@ -204,41 +200,50 @@ public void updateBridge() {
         }
         logger.debug("[updateBridge] Updating bridge");
 
-        // Call receiveHeaders
-        try {
-            int numberOfBlocksSent = updateBridgeBtcBlockchain();
-            logger.debug("[updateBridge] Updated bridge blockchain with {} blocks", numberOfBlocksSent);
-        } catch (Exception e) {
-            logger.error(e.getMessage(), e);
-            panicProcessor.panic("btclock", e.getMessage());
+        if(shouldUpdateBridgeBtcBlockchain) {
```

Comment:
nit: for the if conditions
```suggestion
        if (shouldUpdateBridgeBtcBlockchain) {
```